### ISSUE TITLE
Since v1.2.2, running `ng test` with source maps enabled (default) makes some bundles be loaded two times more through XHR under Chrome only

### ISSUE THREAD

**Issue by ymeine** [angular/angular-cli#7525]
### Bug Report or Feature Request (mark with an `x`)

- [x] bug report -> please search issues before submitting
- [ ] feature request

### Versions.

```
@angular/cli: 1.3.1
node: 8.4.0
os: win32 x64
@angular/animations: 4.3.5
@angular/common: 4.3.5
@angular/compiler: 4.3.5
@angular/core: 4.3.5
@angular/forms: 4.3.5
@angular/http: 4.3.5
@angular/platform-browser: 4.3.5
@angular/platform-browser-dynamic: 4.3.5
@angular/router: 4.3.5
@angular/cli: 1.3.1
@angular/compiler-cli: 4.3.5
@angular/language-service: 4.3.5
```

### Repro steps.

To reproduce the issue, run `ng test` on a project freshly generated by the CLI and make sure `--sourcemap` is enabled and that you are testing it with Chrome. Then open the developer tools and check the network panel.

The following files are all loaded three times (once through a script inclusion and then twice through XHR): 

- `inline.bundle.js`
- `vendor.bundle.js`
- `main.bundle.js`

These are all bundles. Those additional bundles though don't suffer from this problem: 

- `polyfills.bundle.js`
- `scripts.bundle.js`

### The log given by the failure.

![2017-08-28 15_01_07-karma](https://user-images.githubusercontent.com/2077669/29774509-ec22d11c-8c01-11e7-8aae-3705ffc99ee8.png)

### Desired functionality.

The three bundles above should be loaded only once, through script inclusion, and not at all through XHR.

### Mention any other details that might be useful.

I investigated a bit more and identified commit `19f04af0777391cc72d096a0e315fe87bab161c6` as the one introducing the bug. The content changed in this commit is fairly simple, and you'll notice it concerns the source maps feature. I quickly tested removing those lines in `packages/@angular/cli/plugins/karma.ts`: 

- [73](https://github.com/angular/angular-cli/blob/19f04af0777391cc72d096a0e315fe87bab161c6/packages/%40angular/cli/plugins/karma.ts#L73): resolves path to newly added dependencies `karma-source-map-support`
- [76-77](https://github.com/angular/angular-cli/blob/19f04af0777391cc72d096a0e315fe87bab161c6/packages/%40angular/cli/plugins/karma.ts#L76-L77): uses the previously resolved path to actually add files into the Karma configuration

After removing those lines the bug disappears (even if I guess the source maps feature doesn't work a s expected after that.)

**Comment by ludvon** [comment]
Might cause @7296?

**Comment by clydin** [comment]
The CLI has been updated significantly since version 1.3.1.  The current stable version is 8.3.5.  If this problem is still being encountered with the current version, please consider opening a new issue with updated details.

**Comment by jneuhaus20** [comment]
This is still present in 8.3.6.  Can we just reopen this bug? It's already triaged, has activity on it, and (arguably most importantly) this is where google brought me when I searched the issue.
